clang/lib/DPCT/Asm/AsmLexer.cpp

```diff
@@ -196,6 +196,11 @@ bool InlineAsmLexer::lexTokenInternal(InlineAsmToken &Result) {
       ++CurPtr;
       Kind = asmtok::pipe;
     } else {
+      // PTX allows % builtin-identifier.
+      if (isIdentifierContinue(Char)) {
```

Comment:
Should better merge this line to the pre line? `else if (isIdentifierContinue(Char))`

Reply:
they are on different paths

Reply:
done